### PR TITLE
dengisend.ru

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: src/chrome/content/rules/Dengisend.ru.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	<target host="kamkombank.dengisend.ru" />
+	<target host="kibank.dengisend.ru" />
+	<target host="rt-iframe.dengisend.ru" />
+	<target host="bank-enisey.dengisend.ru" />
+	<target host="vyborg-bank.dengisend.ru" />
+	<target host="sputnik.dengisend.ru" />
+	<target host="onlime.dengisend.ru" />
+	<target host="pin-card.dengisend.ru" />
+	<target host="slavcred.dengisend.ru" />
+	<target host="nsbank.dengisend.ru" />
```
Comment (J0WI) [inline src/chrome/content/rules/Dengisend.ru.xml]: clcorp.
rtsbank.

**Comment by J0WI** [comment]
Please also sort the subdomain hosts in alphabetical order.

**Comment by galeksandrp** [comment]
Added.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**